### ISSUE TITLE
Feature request: theme pull and theme push to have a --live (-l) option to select the current live theme

### ISSUE THREAD

**Issue by ADTC** [Shopify/shopify-cli#1647]
# Feature request

A new option in the `theme push` and `theme pull` options to automatically select the current live theme.

The option could be referenced with `--live` or `-l` as a short form.

## Expected behavior

It should pull or push the current live theme.

In case of `theme push` the option would only select the live theme, but will not push unless the user confirms. Adding the `--allow-live` (`-a`) option would push without user confirmation.

## Actual behavior

There's no such option right now. `-i live` did not work.

**Comment by ADTC** [comment]
@karreiro thank you so much! It's great to see my feature request implemented. I'm just a little worried about --live implying --allow-live, as originally I had only imagined this option to pre-select the live theme but still ask for confirmation (unless --allow-live also given).

**Comment by karreiro** [comment]
Thanks for the feedback @ADTC! The workflow you've mentioned makes total sense. I just submitted the change not to imply `--allow-live`.

Also, I've noticed the release process to 2.7.2 has started:

<img width="1301" alt="image" src="https://user-images.githubusercontent.com/1079279/144094610-5baaa921-b590-4882-8189-c153888aaccf.png">

So, it's important to notice that the 2.7.2 version of the CLI **is going imply** `--allow-live`. But, on our next release, the fix you've suggested https://github.com/Shopify/shopify-cli/pull/1807 will be incorporated, and `--allow-live` won't be implied anymore.

Thanks again for the feedback 🚀
